### DISCUSSION TITLE
GAMMA prepare for Mintpy

### DISCUSSION THREAD

**Discussion by gaiyf** [insarlab/MintPy#1023] [Q&A]
Hello everyone, I am a newer to GAMMA, and I want seek your help with GAMMA scripts for the preparation of the Mintpy process.
Best regards.

**Comment by scottstanie** [comment]
Have you checked out the existing [prep_gamma.py](https://github.com/insarlab/MintPy/blob/main/src/mintpy/prep_gamma.py) script? If so, was there something that didn't work for you?
